app/src/main/java/org/p2p/wallet/updates/SocketUpdatesManager.kt

```diff
@@ -33,7 +33,7 @@ import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManag
 
 private const val DELAY_MS = 5000L
 
-private const val TAG = "Sockets:SocketUpdatesManager"
```

Comment:
What's the difference between `Sockets:SocketUpdatesManager` and `SocketUpdatesManager`?
I removed prefix because it doesn't fit in logs. 
It says `Soc........esManager` and it loses all it's value